priv/ie_gen_v1.erl

```diff
@@ -1339,7 +1341,8 @@ msgs() ->
 	 {"BAR ID",				   'O', {'X', '-', '-', 'X'}},
 	 {"Redundant Transmission Parameters",	   'C', {'-', '-', '-', 'X'},
 	  [{"F-TEID",				   'M', {'-', '-', '-', 'X'}},
-	   {"Network Instance",			   'C', {'-', '-', '-', 'X'}}]}]},
+	   {"Network Instance",			   'C', {'-', '-', '-', 'X'}}]},
+	 {"IPFIX Policy",		   'O', {'X', 'X', 'X', 'X'}}]},
```

Comment:
Sorry, I don't see what is wrong here. As I see I didn't change indentation,  the existing lines and the new one uses the same as the others(one tab at the beginning of the line and few spaces after) . IMHO we should avoid using tabs because the actual display depends on the editor settings, but that is a long term desire again.

Reply:
I have stated it many times, a TAB is 8 SPACEs. That is the default that all the Erlang/OTP system code used for years.
The indentation style is also what the OTP code has been using for years and it codified in the EMACS Erlang mode.

Reply:
The distance between the IE name and the flags is wrong

Reply:
I'd say we can use another indentation default if it suits us better in our products. I don't think the OTP team will be mad at some people using Erlang for not respecting their own code conventions. I personally think mixing tabs and spaces in indentation is a bad idea.

That said, we'll use this convention for the moment then. But let's open an issue; I'm open to both tabs and spaces, but mixing them is really weird.

Reply:
I am also open to discuss this.

Reply:
The current style suites me just fine, otherwise I wouldn't have chosen it.

We can use whatever new style we want for new projects. But the existing projects stay the way they are.

Reply:
indentation fixed